source/VideoTransportSlider.js

```diff
@@ -646,10 +646,12 @@
 		* @private
 		*/
 		setValue: function(val) {
-			if(Math.abs(this.calcVariationPercent(val)) > this.smallVariation) {
+			var v = this.clampValue(this.min, this.max, val);
+			this.currentTime = v;
```

Comment:
Please disregard the previous comment, forgot we weren't using the knob value. :)

Could we do away with the explicit clamping and move a modified version of this line to the end of the function:
    `this.currentTime = `this.value`

It seems that all code paths in this function will eventually clamp and set `this.value`.